images/backend.json

```diff
@@ -1,7 +1,7 @@
 {
     "hostname": "{{env `PERM_ENV` }}",
     "instance_type": "m4.large",
-    "volume_size": "100",
+    "volume_size": "1000",
```

Comment:
Ideally, this change would be in a separate commit, since it's unrelated to the security group changes.